doc/source/timeseries.rst

```diff
@@ -889,9 +889,29 @@ The key features of a ``DateOffset`` object are:
 * It can be multiplied by an integer (positive or negative) so that the
   increment will be applied multiple times.
 * It has :meth:`~pandas.DateOffset.rollforward` and
-  :meth:`~pandas.DateOffset.rollback` methods for moving a date forward or 
+  :meth:`~pandas.DateOffset.rollback` methods for moving a date forward or
   backward to the next or previous "offset date".
 
+.. note::
+
+    Beware that adding an offset to a ``DatetimeIndexes`` with
+    consecutive values can produce a ``DatetimeIndexes`` with gaps, for example:
+
+    .. ipython:: python
+
+    index = pd.DatetimeIndex(start='2018-02-27', periods=3, freq='D')
+    index
+    index + pd.tseries.offsets.DateOffset(months=4, days=2)
+
+A way to achieve ``DatetimeIndexes`` without gaps is to shift the first date
```

Comment:
Could you also make sure the rest of this new section is also contained in the same `.. note::` directive (as it continues the point made above)?

After that LGTM.